### PR TITLE
feat: support `create/alter/drop user` statement

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by dl239** [comment]
@tobegit3hub will add user relevant information to `openmldb-batch` in another pr

File: src/sdk/sql_cluster_router.cc [hunk 1]
```diff
@@ -2843,6 +2819,10 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteSQL(
                 ::openmldb::base::Status base_status;
                 if (is_online_mode) {
                     // Handle in online mode
+                    config.emplace("user", GetRouterOptions()->user);
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: rename to `spark.openmldb.user`

File: src/sdk/sql_cluster_router.cc [hunk 2]
```diff
@@ -2843,6 +2819,10 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteSQL(
...
+                    if (!GetRouterOptions()->password.empty()) {
+                        config.emplace("password", GetRouterOptions()->password);
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: ename to `spark.openmldb.password`

File: src/sdk/sql_cluster_router.cc [hunk 3]
```diff
@@ -2843,6 +2819,10 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteSQL(
...
+                    }
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: add to all job task config

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by lqy222 — APPROVED** [review]

**Pull request merged**